interfaces/builtin/desktop.go

```diff
@@ -205,8 +205,8 @@ dbus (send)
 
 ## Allow access to xdg-document-portal file system.  Access control is
 ## handled by bind mounting a snap-specific sub-tree to this location.
-owner /run/user/[0-9]*/doc/ r,
-owner /run/user/[0-9]*/doc/** rw,
+owner /run/user/[0-9]*/doc/{,*} r,
+/run/user/[0-9]*/doc/*/** rw,
```

Comment:
This looks good, but can we also have a clarifying comment:
```
# Allow rw access without owner match to the documents themselves since
# the user guided the access and can specify anything DAC allows.
```